azure-pipelines.yml

```
@@ -26,4 +26,10 @@ steps:
   inputs:
     testResultsFormat: 'JUnit'
     testResultsFiles: 'util/junit.xml'
-    testRunTitle: 'Run unit tests'
\ No newline at end of file
+    testRunTitle: 'Run unit tests'
+
+- task: RichCodeNavIndexer@0
+  inputs:
+    serviceConnection: 'rich-code-navigation'
+    githubServiceConnection: Microsoft
+    languages: typescript

```

I would suggest using our internal feed for this repository so it will get the latest bits as early as possible.

```suggestion
    languages: typescript
    serviceEndpoint: https://int.richcodenav.vsengsaas.visualstudio.com
```